src/avtaleland.js

```diff
@@ -20,32 +20,60 @@ const avtaleland = [
     term: 'Chile',
   },
   {
-    kode: 'IN',
-    term: 'India',
+    kode: 'FR',
+    term: 'Frankrike',
   },
   {
     kode: 'IL',
     term: 'Israel',
   },
+  {
+    kode: 'IM',
+    term: 'Isle of Man'
+  },
+  {
+    kode: 'IN',
+    term: 'India',
+  },
+  {
+    kode: 'IT',
+    term: 'Italia',
+  },
+  {
+    kode: 'JE',
+    term: 'Jersey'
+  },
+  {
+    kode: 'HR',
+    term: 'Kroatia',
+  },
   {
     kode: 'ME',
     term: 'Montenegro',
   },
+  {
+    kode: 'PT',
+    term: 'Portugal',
+  },
+  {
+    kode: 'CA_QUEBEC',
```

Comment:
CA-QC ser ut til å være en standard ifølge https://en.wikipedia.org/wiki/ISO_3166-2:CA

Reply:
gikk for CA_QC siden bindestrek skapte issues når man prøvde å konvertere til java.